chutney/ui/src/app/modules/scenarios/components/execution/detail/execution.component.ts

```diff
@@ -536,7 +536,6 @@ export class ScenarioExecutionComponent implements OnInit, OnDestroy, AfterViewI
         if (scrollIntoView && step) {
             document.getElementById(step['rowId']).scrollIntoView({behavior: 'smooth', block: 'start'});
         }
-        this.elementRef.nativeElement.scrollIntoView({behavior: 'instant', block: 'start'})
```

Comment:
If you add this after the deleted line (keeping it), it works as planned :
`
findScrollContainer(this.elementRef.nativeElement).scrollBy(0, -60); // 60 < (report-header + report-header-margin-bottom + ctx-var-raw-margin-bottom) = 55
`
Tell me if you're ok with this fix.